agentstack/generation/agent_generation.py

```diff
@@ -21,7 +21,7 @@ def generate_agent(
     if not backstory:
         backstory = 'Add your backstory here'
     if not llm:
-        llm = 'Add your llm here with format provider/model'
+        llm = 'openai/gpt-4o'
```

Comment:
We could store this in agentstack.json for future reference.

Reply:
like which provider/model they prefer?